### PR TITLE
Add `OutputType.Minimal`

### LINKED ISSUE #184: new Minimal OutputType
#### Is the feature request related to a problem

No.

#### Describe the solution

#71 added a new `OutputType.Compact` which greatly reduces the size of the output compared to the default `Full`.
When I use `git diff` I often pass `-U0` to _only_ see the changed lines.
To get a similar output for our [API approval tests](https://github.com/fluentassertions/fluentassertions/blob/develop/Tests/Approval.Tests/ApiApproval.cs) in Fluent Assertions I wrote a custom string comparer to simply skip all unchanged lines.

This is a proposal to add a new `OutputType`, I suggest `Minimal`, which is identical to `Full` except that it excludes unchanged lines.

#### Describe alternatives considered

Write my own custom string comparer

### PR DESCRIPTION
**Pull request by jnyrup**
This fixes #184.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -130,6 +130,15 @@ This output type gives the most information, but if verified files are long, it
 7 Seventh line
 ```
 
+Lastly, there is `OutputType.Minimal` which will show only the changed lines. Here is how the `Full` output looks like in `Minimal` mode.
```
Comment (0xced) [inline readme.md:133]: Given that `Full` is a valid value of the `OutputType` enum, I'd reword it like that to reduce possible confusion:

> Here is how the complete output looks like in `Minimal` mode.

Comment (0xced) [reply readme.md:133]: (or **entire** instead of **complete**, can't decide which one is better)

**Comment by SimonCropp** [comment]
thanks. this is deployed
